Objective-C/CBLReplicator.mm

```diff
@@ -301,17 +300,13 @@ - (void) stop {
 }
 
 - (void) stopped {
```

Comment:
hmmm... this looks important.  What is it doing?

Reply:
It's just a method to set the stopped state. There are a couple of places calling that to set the replicator in stopped state.

Reply:
Yah... ok.  Makes perfect sense, now.  Don't know what I was thinking...